src/libraries/Common/src/System/Security/Cryptography/RSASecurityTransforms.cs

```diff
@@ -99,10 +99,12 @@ public override RSAParameters ExportParameters(bool includePrivateParameters)
                     throw new CryptographicException(SR.Cryptography_OpenInvalidHandle);
                 }
 
+#pragma warning disable CA1416 // https://github.com/dotnet/runtime/issues/51098
```

Comment:
I'm not very happy about all these pragmas they are not really helping us to improve the dev experience.

Reply:
yep, but what is a better alternative here?

Reply:
This API is unsupported so we should annotate the public base class (RSA type) methods are unsupported

Reply:
> we should annotate the public base class (RSA type) methods are unsupported

That's where it gets tricky.  It's an abstract class.  Doesn't work out of the box (though it can/should before .NET 6 releases), but anyone can write one.

I feel like if we mark anything in the base class, it would just be the Create() method.

Reply:
> That's where it gets tricky. It's an abstract class. Doesn't work out of the box (though it can/should before .NET 6 releases), but anyone can write one.

I think that would be very rare and would actually misfire warning only if someone writes implementation which works on iOS and call it from iOS-specific app/library.

Reply:
Annotated the public base class (RSA type) and derived class (RSASecurityTransforms) methods as unsupported. Annotating only the base class methods didn't work for me. As for some private/internal methods I still leave warning suppression.